lib/quantum/parser.ex

```diff
@@ -2,15 +2,15 @@ defmodule Quantum.Parser do
 
   @moduledoc false
 
-  def parse("*/" <> i, min, max) do
```

Comment:
The warnings were:
"Parameter `max` has same name as the `Kernel.max` function." and
"Parameter `min` has same name as the `Kernel.min` function"

I agree that changing min/max to a/b does not improve readability.

However, looking at the changes it seems that we could replace `a, b` nearly everywhere with a better named `range` and thus (1) improve readability and (2) reduce number of arguments. I'll give it a try tomorrow.

Thanks for asking! :-)